src/include/actionClosures.js

```diff
@@ -162,8 +162,11 @@ WMEAC.removeClosure = function (closures, successHandler, failureHandler)
     var cab = require("Waze/Modules/Closures/Models/ClosureActionBuilder");
     var sc = require("Waze/Modules/Closures/Models/SharedClosure");
     var t = {};
-    var c = new sc({closures: [].concat(closures)}, {dataModel: W.model, segmentSelection: W.selectionManager.getSegmentSelection(), isNew: true});
-    t.actions=[cab.delete(c)];
+    let segs = WMEAC.segmentsIDsToSegments(closures.map(closure => closure.attributes.segID));
+    segs = segs.filter(function (seg) {
+        return seg.isAllowed(seg.permissionFlags.EDIT_CLOSURES);
+    });
```

Comment:
These 4 lines also appear in removeInWME() function. Is there a reason they are needed here also?

Reply:
It is not totally clear if I could pass the `segs` variable from `removeInWME` or if I had to refetch them here. We could compare `this.segIDs` in `removeInWME` and `closures.map(closure => closure.attributes.segID)` here. 

We need them in `new sc({segments: segs,`.

I had only my example and couldn't be sure that it was the same in all cases.

Reply:
OK, looks like we just need the first line, but the other 3 are not needed because any segments that user doesnt have permission for have already been filtered out. I will accept this and remove those 3 lines before I build the wme-ac.js file.

Reply:
Ok fine!

Thanks.